### ISSUE TITLE
Ingestion Request [07-20-2024]: CCL 2024

### ISSUE THREAD

**Issue by tianzhiliang** [acl-org/acl-anthology#3117]
### General information about this request

- [X] I confirm that I have read the [Information for Submitters](https://aclanthology.org/info/contrib/).
- [X] I am submitting a request for a **new venue** that does not exist in the ACL Anthology yet.

### Venue Identifier

CCL

### Volume Title

Proceedings of the 23rd China National Conference on Computational Linguistics

### Venue Website (only if you are submitting a new venue)

http://cips-cl.org/static/CCL2024/index.html

### Date of Publication

2024-07-20

### Supporting Information

This event will include following volumes ({volume identifier}; {volume name}) .
●2024.CCL-paper; Proceedings of the 23rd China National Conference on Computational Linguistics
●2024.CCL-demo; Proceedings of the 23rd China National Conference on Computational Linguistics: System Demonstrations
●2024.CCL-tutorials; Proceedings of the 23rd China National Conference on Computational Linguistics: Tutorial Abstracts
●2024.CCL-evaluations; Proceedings of the 23rd China National Conference on Computational Linguistics: Evaluations
●2024.CCL-forum; Proceedings of the 23rd China National Conference on Computational Linguistics: Frontier Forum
●2024.CCL-findings; Findings of the 23rd China National Conference on Computational Linguistics

**Comment by anthology-assist** [comment]
@tianzhiliang thank you for submitting the request! Please allow two weeks for ingestion once ingestion material is submitted.

**Comment by tianzhiliang** [comment]
Dear Madam or Sir,
         We have submitted the request to publish the CCL-2024's materials on ACL Anthology. Now, the materials are ready for publishing. Here are the materials (https://drive.google.com/file/d/17oSq48VKi24smEwyQ_dBIt4Z2mMKpSd5/view?usp=drive_link). Please feel free to contact me.

**Comment by anthology-assist** [comment]
@tianzhiliang Could you grant me access to the google drive? anthologygit@gmail.com

**Comment by tianzhiliang** [comment]
Sure. I have granted the access to this link. Please try it again. Please feel free to contact me. Thank you so much!

> @tianzhiliang Could you grant me access to the google drive? [anthologygit@gmail.com](mailto:anthologygit@gmail.com)

**Comment by tianzhiliang** [comment]
> @tianzhiliang Could you grant me access to the google drive? [anthologygit@gmail.com](mailto:anthologygit@gmail.com)

I have granted this link. Can you access it? Is this issue under-processing?

**Comment by mjpost** [comment]
Hi @tianzhiliang, can you please share the link here?

**Comment by tianzhiliang** [comment]
> Hi @tianzhiliang, can you please share the link here?

https://drive.google.com/file/d/17oSq48VKi24smEwyQ_dBIt4Z2mMKpSd5/view?usp=drive_link

**Comment by mjpost** [comment]
Ping @anthology-assist

**Comment by tianzhiliang** [comment]
@anthology-assist https://drive.google.com/file/d/17oSq48VKi24smEwyQ_dBIt4Z2mMKpSd5/view?usp=drive_link

This is the link for the materials.

**Comment by tianzhiliang** [comment]
I am writing to confirm wether this issue is under processing? We have submit the request and materials for 5 monthes.

**Comment by anthology-assist** [comment]
@tianzhiliang Sorry about the late response. I confirm that this is under processing and should be done soon.

**Comment by mjpost** [comment]
To add information to this, we are experiencing a problem. In the past, we could read BibTeX files with multiple author lines, which let us handle their main names and variants. However, this functionality no longer works; it appears that multiple author lines is now a parsing error. So we are looking for a workaround to incorporate the Han name variants.

**Comment by mjpost** [comment]
@anthology-assist A simple way to handle the name variants issue is to downgrade pybtex to an older version, one that worked in the past

**Comment by mjpost** [comment]
Okay, I realized the problem. Last year, we ran into the same error (https://github.com/acl-org/acl-anthology/pull/2487). The solution was a temporary downgrade to pybtex==0.22.2.

I cobbled together a manual fix. A note to us and to CCL submitters: next time, please reformat your proceedings to use [aclpub2](https://github.com/rycolab/aclpub2/). Going forward, I think we will only support name variants that way.

**Comment by tianzhiliang** [comment]
Dear mjpost and anthology-assist ,
      I see the issue was closed on Github. Does it mean CCL 2024 has been published successfully? We are hoping for the publishing since we had waited for more than half an year. @anthology-assist  @mjpost 

Yours,
Zhiliang

**Comment by mjpost** [comment]
Yes, this has been live for many months: https://aclanthology.org/venues/ccl/
